v2/oauth/common-actions.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Common Actions
+hide_title: true
+---
+
+# Common Actions
```

Comment:
This page can get very long. Instead, we should have a section called "Customizations" in the docs, which contains pages like:
- Working with scopes
- Adding custom claims in the OAuth access or ID token
- Verifying the OAuth access token and ID token